projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -275,13 +278,21 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
         const selectedItems: SelectOptionComponent<V>[] = options.filter(item => this.isSelectedItem(item));
 
         return {
-          label: selectedItems.length === 0 ? this.placeholder : selectedItems[0]?.label,
+          label: this.getLabel(selectedItems),
           selectedItemsCount: selectedItems.length
         };
       })
     );
   }
 
+  private getLabel(selectedItems: SelectOptionComponent<V>[]): string {
+    if (selectedItems.length === 0) {
+      return this.placeholder === undefined ? '' : this.placeholder;
+    }
+
+    return this.prefix === undefined ? selectedItems[0]?.label : `${this.prefix}${selectedItems[0]?.label}`.trim();
```

Comment:
oooh.. don't you think this is confusing? Currently, we should placeholder, something like "Select labels" and once we select few labels, we would show "label 1 + 2" others. Now we are showing that upfront. IMO, we lose the distinction and it will be hard to distinguish between your new placeholder and existing selection option trigger

Reply:
Hmm. Fair point. There are probably a few uses that we don't want to follow this pattern. Let me make it configurable.

Reply:
Ah, so I just looked back at the code and I had the foresight to already make this configurable. If the user doesn't see the `prefix` input, the behavior should be the same as before this PR.